### ISSUE TITLE
Variable order gets mixed up sometimes

### ISSUE THREAD

**Issue by Eschon** [babel-plugins/babel-plugin-merge-sibling-variables#1]
I have the following variable declarations:

```JavaScript
var map = this._map;
var layerGroup = this.layerGroup;
var iconSize = this.iconSize;
var iconAnchor = [ iconSize[0] / 2, iconSize[1] / 2 ];
var markers = this.liftMarker.all.concat(this.slopeMarker.all);
var invisible = [];
```

The output when using this babel-plugin is:

```JavaScript
var map = this._map, layerGroup = this.layerGroup, iconAnchor = [iconSize[0] / 2, iconSize[1] / 2], invisible = [];
var iconSize = this.iconSize, markers = this.liftMarker.all.concat(this.slopeMarker.all);
```

As you can see `iconSize` is now declared after `iconAnchor` which produces an error.
If I put something after the declaration of `iconSize` that stops it from being siblings with the rest it works:

```JavaScript
var map = this._map;
var layerGroup = this.layerGroup;
var iconSize = this.iconSize;
iconSize;
var iconAnchor = [ iconSize[0] / 2, iconSize[1] / 2 ];
var markers = this.liftMarker.all.concat(this.slopeMarker.all);
var invisible = [];
```

**Comment by sebmck** [comment]
Fixed as of v1.0.2, thanks :blush:

**Comment by Eschon** [comment]
As far as I can tell it works now.
Thanks for the quick fix.
